docs/Migration-of-Spaces.md

```diff
@@ -0,0 +1,21 @@
+# Migration of Spaces
+
+Starting with the planned future GA Brim release `v0.25.0`, imported data will
+now be stored in "Data Pools" in a "Zed Lake" rather than "Spaces" as they were
+previously.
+
+A [design document](https://github.com/brimdata/zed/blob/main/docs/lake/design.md)
+provides a thorough overview of pools and how they work. In brief, the use of
+pools ultimately enables new functionality that was not previously possible in
+Brim, including:
+
+* Data may be incrementally added to, or deleted from, a pool.
+* Zed queries/analytics can make use of data stored across multiple pools.
+* Search indexes can be created to accelerate query performance.
+* Pre-computed derived analytics can be leveraged to accelerate computations on stored data.
+
+When upgrading to GA Brim release `v0.25.0` or newer, any data you had stored
+in Spaces that you wish to continue accessing will need to be migrated to
+pools. The tooling to perform this migration is still under development. This
```

Comment:
I see "Spaces" was capitalized but "pools" was lower case. Not sure which is correct but they seem on the same level.

Reply:
Good point @jameskerr. I just pushed a commit to capitalize the "Pool" references also. Merging!